Makefile

```diff
@@ -256,8 +256,8 @@ docker-builder:
 
 build-in-docker: clean docker-builder
 	docker run \
-		-v `pwd`:/gopath/src/k8s.io/node-problem-detector/ npd-builder:latest bash \
-		-c 'cd /gopath/src/k8s.io/node-problem-detector/ && make build-binaries'
+		-v `pwd`:/node-problem-detector/ npd-builder:latest bash \
+		-c 'cd /node-problem-detector/ && make build-binaries'
```

Comment:
It could be confusing to put the project that use `go module` under `/gopath/...`

Reply:
I like this change. Can you verify that the `chown` is similar to the gopath for a docker build? I think it might adjust it to root as compared to the `/gopath` directory.